### ISSUE TITLE
Can't save session -  Error "busy- close file dialog box to resume" 

### ISSUE THREAD

**Issue by Speqtrvm** [brunoherbelin/vimix#92]
I am using Vimix 0.8.0 on Mac M1 12.5.1 Monterey.

When I try to save a session it will flash up the message "busy- close file dialog box to resume" for a nanosecond and then just resume without saving. 

I tried reinstalling and also using Vimix 0.7.3  instead but the issue remains.

Thanks in advance!

**Comment by brunoherbelin** [comment]
humm... strange and could not be replicated (I have both a Mac on BigSur and one on Ventura; all works fine there).
Could it be a restriction of Privacy and Security from OSX for 'external' apps ? (i.e. preventing to access harddrives)

**Comment by Speqtrvm** [comment]
I gave Vimix full disk access so that it also can access files and folders with no restrictions and restarted but it's still the same. No idea what it is but it works on my intel mac so I figured it was an M1/M2 problem.

**Comment by brunoherbelin** [comment]
I had another report of a similar issue : it occurred when running a new version of vimix after install. 
This can be fixed by resetting the settings of vimix.

Instructions to reset settings in finder; 
do "Go to folder" (shift+command+G)
enter "/Users/[username]/Library/Application Support/vimix"  (with your username)
delete the folder 'vimix'

Thanks for confirming that this is it

**Comment by Speqtrvm** [comment]
Just tried it and deleted the whole folder with the XML file and everything in there but didn't solve the problem unfortunately. Thanks for looking into it.

**Comment by brunoherbelin** [comment]
humm... and this happens also with the [new package 0.8.2](https://github.com/brunoherbelin/vimix/releases/tag/0.8.2b) ?

**Comment by Speqtrvm** [comment]
Just installed 0.8.2. and copied the same procedure. Still not able to load or save a session : (

Same computer setup as original message in the thread.

**Comment by brunoherbelin** [comment]
Thanks for not surrendering with vimix! I hope we'll figure this out !

Can you please try to run this command in a terminal;

```
osascript  -e 'try' -e 'POSIX path of ( choose file name with prompt "Save Session" default location "/Users/" )' -e 'on error number -128' -e 'end try'
```

.. and report back on the success or error messages? 
(i.e. on success, if you select a path and clic 'save', it should return the filename)

**Comment by Speqtrvm** [comment]
yes it returned the path followed by the file name I entered. However when i went into the pathway folder there was no session file created there.
